src/views/product-tutorials-view/server.ts

```diff
@@ -1,26 +1,35 @@
+import { ProductData } from 'types/products'
 import { getAllCollections } from 'lib/learn-client/api/collection'
 import { getProduct } from 'lib/learn-client/api/product'
 import {
   Collection as ClientCollection,
   ProductOption,
-  Product as ClientProduct,
 } from 'lib/learn-client/types'
 import { stripUndefinedProperties } from 'lib/strip-undefined-props'
 import { filterCollections } from './helpers'
 
 export interface ProductTutorialsPageProps {
   collections: ClientCollection[]
-  product: ClientProduct
+  product: ProductData
 }
 
-/** @TODO add sidebar sort capability */
-
+/**
+ * Given a ProductData object (imported from src/data JSON files), fetches and
+ * returns the page props for `/{product}/tutorials` pages.
+ *
+ * Merges the product object fetched from `/products/:identifier` with the given
+ * ProductData object and returns the merged object under the `product` page
+ * prop, which is needed for other areas of the app to function.
```

Comment:
Will let this comment do the initial explaining, and absolutely happy to answer any questions. 💖